docs/install/upgrade.md

```diff
@@ -0,0 +1,37 @@
+# Upgrade
+
+This article walks you through how to upgrade your Coder server.
+
+## Upgrading the Coder server
+
+To upgrade your Coder server, simply reinstall Coder using your original method
+of install.
+
+> Prior to upgrading Coder, we _highly_ recommend taking a database snapshot, as
+> Coder does not support rollbacks.
```

Comment:
Use the danger blockquote here...

```
<blockquote class="danger">
  <p>
  ...
  </p>
</blockquote>
```

E.g:

<img width="990" alt="Screen Shot 2022-08-09 at 8 34 34 PM" src="https://user-images.githubusercontent.com/7416144/183791161-3d97f457-d9c4-4ed5-9246-b5b8d84f9ae2.png">

You can see this laid out in the yarn dev workflow as well.